kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/marshalling/CustomXstreamMarshallerBuilder.java

```diff
@@ -35,7 +35,8 @@ public Marshaller build(Set<Class<?>> classes, MarshallingFormat format, ClassLo
             return new XStreamMarshaller(classes, classLoader) {
                 @Override
                 protected void buildMarshaller(Set<Class<?>> classes, ClassLoader classLoader) {
-                    xstream = XStreamUtils.createNonTrustingXStream(new PureJavaReflectionProvider(), new DomDriver("UTF-8", new XmlFriendlyNameCoder("_-", "_")));
+                    xstream = XStreamUtils.createNonTrustingXStream(new PureJavaReflectionProvider(), new DomDriver("UTF-8", new XmlFriendlyNameCoder("_-", "_"))
+                            , next -> new CustomElementIgnore(next));
```

Comment:
the only way to get a buildMarshaller with this `ignoreUnknownFields` feature is by using and passing this new `CustomElementIgnore` - so I'm just thinking whether it would be worth to make it `public` instead of `protected` - otherwise we are forcing customer to create a package with the same structure - or am I wrong? wdyt?

Reply:
This class is intender to be used only in inherited class, so protected is right